policy/lib/tekton/task.rego

```diff
@@ -126,14 +126,32 @@ _task_params(task) := params if {
 # task_param returns the value of the given parameter in the task.
 task_param(task, name) := _task_params(task)[name]
 
+# slsa v0.2 results
+_task_results(task) := task.results
+
+# slsa v1.0 results
+_task_results(task) := task.status.taskResults
```

Comment:
I don't think this is right. AFAIK, results will appear under `.predicate.runDetails.byproducts`, and I'm not sure if this already includes results from TaskRuns.

Reply:
I think only the pipeline results are in byproducts. This is from the taskRun in resolvedDependencies.